spec/http/1.0/openapi.json

```diff
@@ -430,6 +417,23 @@
           "developer_message": {
             "type": "string",
             "description": "An internal message that will be logged but will not be shown to the user or the AI model"
+          },
+          "missing_requirements": {
```

Comment:
Added this to provide a structured way for the OXP server to let the client know that some requirements are missing.